plugin/mutate/source/dextool/plugin/mutate/backend/database/type.d

```diff
@@ -225,3 +225,17 @@ struct MutantMetaData {
         attr.match!((NoMetadata a) {}, (NoMut a) => put(w, "nomut"));
     }
 }
+
+struct Rationale {
+  string time;
+  string rationale;
+}
+
+// Mutant manually marked in db
+struct MarkedMutant {
+    MutationId mutant_id;
+    Path path;
+    Mutation.Status from_status;
```

Comment:
Why is a `from_status` needed?

Reply:
It was just added for more information, i.e. letting the user know that "the mutant you marked last week was alive, and you marked it killed because of rationale" etc. 

But you are correct, it might not be needed.

Reply:
I would recommend removing it if the following statement holds water :
A user will most likely use the mark mutant to test alive mutants. There are no other case that would be a useful scenario /correct use

Reply:
It doesn't? A user would most likely use markMutant to mark alive mutants killed without modifying the code. There are other useful uses as well. Marking "bad" mutants as either killed or killedByCompiler is completely valid. Even if it might be considered misuse and could be abused, it is up to the user?

I still agree though that it could be removed. So, I will remove it in coming commit. :)